src/plugins/code-block.tsx

```diff
@@ -45,7 +45,8 @@ module.exports = (async () => {
     'groovy',
     'dart',
     'text',
-    'plain'
+    'plain',
+    'sh'
```

Comment:
Adding this because the build kept complaining about now having the `sh` language